docs/components/input-field/index.md

```diff
@@ -0,0 +1,133 @@
+# Input field
+
+Use the input field where you want to be able to have a standard field with a set position for
+- label
+- hint text
+- error text
+
+## Arguments
+
+`label: string`
+- Required
+
+`isDisabled: boolean`
+- Optional 
+
+`hint: string`
+- Optional
+- adds an `id` attribute to the underlying `field.Hint` component. This is linked ot the `aria-describedBy` in the input component.
+
+`error: string`
+- Optional
+- adds an `id` attribute to the underlying `field.Error` component. This is linked to the `aria-describedBy` in the input component.
+
+## Attributes
+
+You can pass HTML attributes with `...attributes`
+
+<div class="mb-4">
+  <Form::InputField
+    @label="Label"
+    @hint="extra information about the field"
+    type="text"
+    placeholder="Example text" 
+  />
+</div>
+
+```hbs template
+<Form::InputField
+  @label="Label"
+  @hint="extra information about the field"
+  type="text"
+  placeholder="Example text" 
+/>
+```
+
+
+## Errors
+
+Use `@error` to add an error message for the input field.
+
+<div class="mb-4">
+  <Form::InputField
+    @label="Label"
+    @hint="extra information about the field"
+    type="text"
+    @error="This field is required" 
+  />
+</div>
+
+```hbs template
+<Form::InputField
+  @label="Label"
+  @hint="extra information about the field"
+  type="text"
+  @error="This field is required" 
+/>
+```
+
+
+## isDisabled
+
+Use `@isDisabled` to disable the entire InputField.
+
+<div class="mb-4">
+  <Form::InputField
+    @label="Label"
+    @hint="extra information about the field"
+    type="text"
+    @isDisabled={{true}}
+  />
+</div>
+
+```hbs template
+<Form::InputField
+  @label="Label"
+  @hint="extra information about the field"
+  type="text"
+  @isDisabled={{true}}
+/>
+```
+
+## UI States
+
+### InputField with Label
```

Comment:
I like the named headings here for UI states.  I'll update my PR to follow!

(For others: @nicolechung and I brainstormed how this may be a good idea if we want to add visual regression testing.  Then we can see the component in every expected state and visually diff things)